wire-runtime-swift/src/main/swift/wellknowntypes/Duration.swift

```diff
@@ -81,9 +81,14 @@ public struct Duration {
     public var nanos: Int32
     public var unknownFields: Foundation.Data = .init()
 
-    public init(seconds: Int64, nanos: Int32) {
+    public init(
+        seconds: Int64,
+        nanos: Int32,
+        configure: (inout Self) -> Void = { _ in }
```

Comment:
Is there value in having the `configure` block if all of the fields are required? (Or if there are no fields, like the `OneOfOptions` below?)